providers/azure/resources/azure.lr

```diff
@@ -1960,4 +1962,12 @@ private azure.subscription.policy.assignment @defaults("name enforcementMode") {
   description string
   // Policy enforcement Mode
   enforcementMode string
+}
+
+// Azure IoT Hub Service
+private azure.subscription.iotService {
+  // Subscription identifier
+  subscriptionId string
+  // List of IoT Hubs in the subscription
```

Comment:
```suggestion
  // List of IoT hubs in the subscription
```

Microsoft doesn't capitalize "hub" -- they just capitalize the product name, "IoT Hub"

Reply:
Thanks Letha